README.md

```diff
@@ -0,0 +1,32 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> My Portofolio
```

Comment:
Great job @Oscarj-8 🤝

- [ ] Kindly add a Project Name and Project description to your Readme file. This is because a Readme file is the first document that users of your repo visit.